src/components/todo.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+// import { Loader } from './todoLoader';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  loading: boolean;
+  errorMessage: string;
+  // isSubmitting: boolean;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  loading,
+  errorMessage,
+  // isSubmitting,
+}) => {
+  return (
+    <>
+      {!loading && !errorMessage && (
```

Comment:
```suggestion
      { (
```
remove conditional rendering here - it shouldn't depend on error message at all

Generally - you should add conditional rendering for the entire component TodoList in such cases

About loader (for each todo) here - you should just add or remove className in the modal overlay, but it doesn't mean you shouldn't render the entire todo